src/apis/candidate.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+const candidateApi = axios.create({
+  baseURL: "https://sumit-back-end-env.herokuapp.com/api/v1",
```

Comment:
As discussed this can be made a generic custom hook which calls API and do error handling as well and returns the 
{error,loading,response} object out of the hook.